src/api/overlays/components/DialogComponent.vue

```diff
@@ -25,16 +25,16 @@
     <div class="c-message__icon" :class="['u-icon-bg-color-' + iconClass]"></div>
     <div class="c-message__text">
       <!-- Uses flex-column -->
-      <div v-if="title" class="c-message__title">
+      <div v-if="title" class="c-message__title" aria-label="Dialog title">
```

Comment:
so i think in this case, we shouldn't be using an aria label if it's not actually interactive

Reply:
I guess ideally these would be actual header and text elements for the implicit roles. Will put a fixme in here